component/core/src/test/java/org/exoplatform/social/core/jpa/storage/IdentityStorageTest.java

```diff
@@ -813,11 +813,12 @@ public void testGetAvatarInputStreamById() throws Exception {
     // within this instruction the profile is created implicitly and it does not have an avatar
     String identityId = identity.getId();
     assertNotNull(identityId);
+
     InputStream stream = identityStorage.getAvatarInputStreamById(identity);
-    assertNull(stream);
+    assertNotNull(stream);
```

Comment:
May be add more tests about Avatar File identifier which shouldn't be null + File name + `isDefaultAvatar` to add in Profile + make sure that avatarLastUpdateTime is updated after renaming an identity of type user.

Reply:
done